cfgov/jinja2/v1/_includes/molecules/big-checkbox.html

```diff
@@ -0,0 +1,71 @@
+{# ==========================================================================
+
+   big_checkbox.render()
+
+   ==========================================================================
+
+   Description: Builds checkboxes fields with a large hit target.
+
+   settings (required):
+
+   settings.legend:     Name of the field.
+
+   settings.fields:     An array of all the checkbox labels.
+
+   options (optional):
+
+   options.required:    Whether the field is required. Defaults to false.
+
+   options.disabled:    Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro render(settings, options) -%}
+    {% from 'macros/util/format/url.html' import slugify as slugify %}
+    {% set id_slug = slugify( settings.legend ) %}
+    <fieldset class="u-reset">
+        <div class="form-l_col
+                    form-l_col-1
+                    u-mb0">
+            <legend class="form-label-header">
+                {{ settings.legend }}
+                {%- if options.required -%}
+                    * <span class="u-visually-hidden">Required</span>
+                {%- endif -%}
+            </legend>
+        </div>
+        {% for field in settings.fields %}
+            {% if loop.length == 2 %}
+                <div class="m-big-checkbox m-big-checkbox__1-2">
```

Comment:
Hmmm... I think that's interesting and you're right it needs to get tweaked.

If we do it that way on the whole fieldset, I think we'd have to come up with another name because it isn't just a checkbox anymore. It's a whole form question. 

I'm also wondering how that pattern does hold up for the text inputs, radio buttons and other stuff to atomize without duplicating too much code.

Right now I'm thinking it might go something like this:

```
.o-form-question
   label.a-form-question_label
  .a-text-input

.o-form-question
  legend.a-form-question_label
 .m-big-checkbox
````

What do you think?